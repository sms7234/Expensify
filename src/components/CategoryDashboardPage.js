import React from 'react';
import CategoryList from './CategoryList';
import CategoryListFilters from './CategoryListFilters';
import CategorySummary from './CategorySummary';

const CategoryDashboardPage = () => (
  <div>
    <CategorySummary />
    <CategoryListFilters />
    <CategoryList />
  </div>
)

export default CategoryDashboardPage;
