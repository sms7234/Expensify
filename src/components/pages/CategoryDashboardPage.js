import React from 'react';
import CategoryList from '../lists/CategoryList';
import CategoryListFilters from '../filters/CategoryListFilters';
import CategorySummary from '../independents/CategorySummary';

const CategoryDashboardPage = () => (
  <div>
    <CategorySummary />
    <CategoryListFilters />
    <CategoryList />
  </div>
)

export default CategoryDashboardPage;
