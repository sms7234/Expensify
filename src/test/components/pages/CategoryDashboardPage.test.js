import React from 'react';
import {shallow} from 'enzyme';
import CategoryDashboardPage from '../../../components/pages/CategoryDashboardPage';

test('should render the dashboard page', () => {
  const wrapper = shallow(<CategoryDashboardPage />);
  expect(wrapper).toMatchSnapshot();
})
