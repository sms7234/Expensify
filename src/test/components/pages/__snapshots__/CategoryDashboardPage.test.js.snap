// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the dashboard page 1`] = `
<div>
  <Connect(CategorySummary) />
  <Connect(CategoryListFilters) />
  <Connect(CategoryList) />
</div>
`;
